### DISCUSSION TITLE
Collapsed Mode (Close button on Tab Previews)

### DISCUSSION THREAD

**Discussion by badcuban** [zen-browser/desktop#4078] [Ideas]
On my “On-the-go” set up I use a laptop without a mouse. I prefer the collapsed mode for the tabs to have more screen real-estate but with the existing mod that adds and X to the Favicon it makes it really easy to accidentally press it and close the tab when switching. It would be amazing if on collapsed mode we could set the tab previews to stay when hovering over them and then putting an X (close) on the preview screen that would close the tab. I understand that the keyboard shortcut is probably faster but a simple close button is more intuitive. This would really help for laptop users that are on the go and do not have a mouse for the middle button close. Thank you for all the awesome work you guys do!

**Comment by Rafagd** [comment]
Also, it would be nice if I could have Ctrl+F4 as the shortcut to close tabs. I've always thought Ctrl+W was a bad shortcut that is very error prone and always made me angry whenever I hit it by accident.

**Comment by natnael-wubet** [comment]
hey any solution yet?
